### ISSUE TITLE
Did you forget to add ".catch" to a promise?

### ISSUE THREAD

**Issue by JohanTh** [xolvio/meteor-cucumber#177]
Hi, the following step is giving me a "Did you forget to add ".catch" to a promise?" error:

```javascript
this.Given(/^I am not logged in$/, function (callback) {
      var self = this;

      self.client
        .executeAsync(function (done) {
          Meteor.call('logOutMirror', function (err, res) {
            done(res); // Don't pass `err` into this as the first argument!
          });
        }, function (err, res) {
          if (_.isNull(res.value)) {
            self.client.call(callback);
          } else {
            callback.fail(new Error("You are logged in"));
          }
        });
    });
```

> Scenario: Logging in      ←[2m←[37m# features\all_scenarios.feature:7←[0m
> [chimp] Detected an unhandledRejection.
> [chimp] Finished running async processes with errors

I guess that adding `.call(callback)` after `executeAsync()` won't give me the result I want.

**Comment by samhatoum** [comment]
you don't need to do `self.client.call(callback);` you can just call `callback`

I'm not sure if that will fix this particular issue. If you can create a reproducible repo I can try to help

**Comment by samhatoum** [comment]
See here: https://github.com/xolvio/meteor-cucumber/issues/172

I think it's the same issue

**Comment by samhatoum** [comment]
Fixed in 0.13.3
